qiling/loader/utils.py

```diff
@@ -111,6 +112,14 @@ def getident():
     return arch, ostype, archendian
 
 def ql_pe_check_archtype(path):
+
+    ftype = magic.from_file(path)
```

Comment:
since we are using libmagic, can we just use libmagic for all file type check ?

Reply:
See commit 4795da3.